builtin/logical/consul/path_roles.go

```diff
@@ -29,13 +29,30 @@ func pathRoles(b *backend) *framework.Path {
 				Description: "Name of the role.",
 			},
 
+			// The "policy" and "token_type" parameters were deprecated in Consul back in version 1.4.
+			// They have been removed from Consul as of version 1.11. Consider removing them here in the future.
```

Comment:
We'll likely want to have a discussion around the timeline for officially deprecating these via announcement, code removal, etc.